layouts/partials/search-box.html

```diff
@@ -0,0 +1,48 @@
+<script async src="{{ site.Params.gcseURL }}"></script>
+<div class="gcse-search"></div>
+
+<style>
+    .gsc-input-box {
```

Comment:
I think adding `font-family: Montserrat` here would make the search input font match the rest of our site!

Reply:
I tried that a lot, but for some reason, even with `!important`, it didn't get overwritten.

Reply:
I see - I pulled changes quickly and seems like adding `font-family: var(--bs-body-font-family);` to [`.gsib_a` here](https://github.com/ubccsss/ubccsss.org/pull/207/files#diff-0f8943c9f076b73daead997c35aed1cbe5b72f3797264e2b92146554c9c40747R29-R32) might work? I guess targeting the main input box isn't specific enough.